settings/benchmarks-suite-jdk21.json

```diff
@@ -286,23 +307,48 @@
     {
       "name": "StrengthReductionBenchmark"
     },
+    {
+      "name": "StringCompareLexBenchmark"
+    },
     {
       "name": "StringConcatenationBenchmark",
       "jmhOpts": "-prof gc"
     },
+    {
+      "name": "StringContainsBenchmark",
+      "jvmArgsAppend": "--enable-preview"
+    },
+    {
+      "name": "StringEqualsBenchmark"
+    },
+    {
+      "name": "StringFormatBenchmark",
+      "jvmArgsAppend": "--enable-preview"
+    },
+    {
+      "name": "StringIndexOfBenchmark"
+    },
     {
       "name": "StringPatternMatcherBenchmark"
     },
     {
       "name": "StringPatternSplitBenchmark",
       "jmhOpts": "-prof gc"
     },
+    {
+      "name": "StringReplaceBenchmark"
+    },
+    {
+      "name": "StringSubstringBenchmark"
+    },
     {
       "name": "StringSwitchBenchmark"
     },
     {
-      "name": "TailRecursionBenchmark",
-      "jvmArgsAppend": "-Xss64M"
```

Comment:
Same question

Reply:
Same rationale +

TailRecursionBenchmark has this already declared in the class
@Fork(value = 5, jvmArgsAppend = {"-Xss64M"})
so it was duplicated.